res/css/views/settings/tabs/user/_LabsUserSettingsTab.scss

```diff
@@ -20,6 +20,13 @@ limitations under the License.
 
         .mx_SettingsFlag {
             margin-right: 0; // remove right margin to align with beta cards
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            
+        .mx_ToggleSwitch {
+            float: unset;
```

Comment:
```suggestion
            .mx_ToggleSwitch {
                float: unset;
```

Reply:
> 

Oh! got it, is there a way to know the styling errors beforehand?

Reply:
Run the style linter, `yarn run lint:style`